packages/web/src/app/AudiusQueryProvider.tsx

```diff
@@ -1,34 +1,35 @@
 import { ReactNode } from 'react'
 
 import { AudiusQueryContext } from '@audius/common'
+import { useDispatch } from 'react-redux'
 
 import { apiClient } from 'services/audius-api-client'
 import { audiusBackendInstance } from 'services/audius-backend/audius-backend-instance'
 import { audiusSdk } from 'services/audius-sdk'
 import { env } from 'services/env'
 import { remoteConfigInstance } from 'services/remote-config/remote-config-instance'
-import { store } from 'store/configureStore'
 import { reportToSentry } from 'store/errors/reportToSentry'
 
 type AudiusQueryProviderProps = {
   children: ReactNode
 }
 
-export const audiusQueryContext = {
```

Comment:
does having this out of the render context cause problems on the server? just curious cause weve directly imported this in sign-up sometimes for convenience

Reply:
The problem was we were importing `store` directly from `store/configureStore` which is no longer possible bc the store is configured in the render context. I updated all of the places we were importing `audiusQueryContext` directly to use the context, for example: https://github.com/AudiusProject/audius-protocol/blob/sk-inf-547-ssr-dual-worker/packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx#L61